Gaijinizer patch/Map191.txt

```diff
@@ -481,84 +516,93 @@ I'm glad if you like it ...?#MTLed
 君が彼女の魔力増幅に協力してくれなければ、計画は恐らく
 君の言うように全員失敗で終わっていただろう。
 > CONTEXT: Map191/events/1/pages/0/339/Dialogue/other5
-Thank you, \\N[1]. If you don't cooperate with
-her magic amplification, the plan is probably
-Everyone would have ended in failure as you say.#MTLed
+You have my gratitude, \\N[1]-kun. If you hadn't
+cooperated in amplifying her magic, the plan
+would have ended in complete failure just as you
+said before.#Ron
```

Comment:
Minor flow:
>_**You have my gratitude, \\\N[1]-kun. If you hadn't
cooperated in amplifying her magic, the plan
would have ended in complete failure.
Just like you said before.**_